src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -108,10 +108,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }))
-
             .OnExecuteCompleted(pr =>
             {
-                notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Id,
+                notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Repository.Owner + "/" + TargetBranch.Repository.Name,
```

Comment:
Oh, that was because the `Id` that was being displayed before meant that the message would show the branch name only, and now it's been change to show the repo owner and name. Since you're working on fixing the names of branches to match the nomenclature we use on .com, I thought it would be useful to also aggregate this PR and make sure what what we show in this message also makes sense - we might also want to show the PR target branch name here as well.